card/lib/card/format.rb

```diff
@@ -224,7 +224,11 @@ def hide_views args
     end
     
     def parse_view_visibility val
-      (val || '').split( /[\s\,]+/ ).map { |view| canonicalize_view view }
+      if val.kind_of? Array
+        val.map { |view| canonicalize_view view }
+      else
+        (val || '').split( /[\s\,]+/ ).map { |view| canonicalize_view view }
```

Comment:
Yes, probably what we want here, and it did not change.  I hope this doesn't need to be redone much, although canonicalize_view isn't that expensive and these lists will rarely be long.  You can DRY up the map since it is the same for both paths.